src/components/InterestForm/index.tsx

```diff
@@ -86,46 +125,51 @@ export default function InterestForm({ caseData }: { caseData: CaseListing }) {
     <FormContainer>
       <H3>Submit Interest</H3>
       {submitted ? (
-        <P>We have received your submission!</P>
+        <P>Your submission has been received!</P>
       ) : (
-        <>
-          <FormQuestion $color={COLORS.greyDark}>
-            What role(s) are you applying for?
-          </FormQuestion>
-          <RadioGroup>
-            {radioOptions.map(option => (
-              <Radio key={option}>
-                <RadioInput
-                  id={`radio${option}`}
-                  type="radio"
-                  name="radioOptions"
-                  value={option}
-                  checked={rolesInterested === option}
-                  onChange={event => setRolesInterested(event.target.value)}
-                />
-                <RadioLabel htmlFor={`radio${option}`}>{option}</RadioLabel>
-              </Radio>
-            ))}
-            {missingInfo && rolesInterested === '' && (
-              <ErrorText>Must select your preferred role</ErrorText>
+        <Flex $gap="30px" $direction="column">
+          {(listingData.listing_type === 'CASE' ||
+            listingData.listing_type === 'INT') && (
+            <DateInput
+              label="What is the earliest date you can contact the client?"
+              required
+              error={getErrorText()}
+              name="startDate"
+              value={startDate}
+              setValue={setStartDate}
+            />
+          )}
+          {listingData.listing_type === 'CASE' && !interpretation && (
+            <RadioGroup
+              name="reason"
+              required
+              value={needsInterpreter}
+              setValue={setNeedsInterpreter}
+              options={['Yes', 'No']}
+              label="Do you need language interpretation help for the client?"
+              error={
+                missingInfo
+                  ? 'Must select whether you need language support'
+                  : ''
+              }
+            />
+          )}
+          <Flex $gap="10px" $direction="column">
+            {formQuestion(
+              'Why are you interested in this case?',
+              listingData.listing_type === 'LCA' ||
+                listingData.listing_type === 'DOC',
             )}
-          </RadioGroup>
-          <DateInput
-            label="What is the earliest date you can contact the client?"
-            error={getErrorText()}
-            name="startDate"
-            value={startDate}
-            setValue={setStartDate}
-          />
-          <FormQuestion $color={COLORS.greyDark}>
-            Why are you interested in this case?
-          </FormQuestion>
-          <FormTextArea
-            id="reason"
-            placeholder="I want to work on this case because..."
-            value={reason}
-            onChange={event => setReason(event.target.value)}
-          />
+            <FormTextArea
```

Comment:
should we use the `TextAreaInput` component?

Reply:
yes! changed!